cloud/main.js

```diff
@@ -1,7 +1,60 @@
 _ = require('underscore');
-var Character = Parse.Object.extend("Character", {
 
+var Elo = {
+  startRank: 1200,
+  K: 20,
+  expected: function(rankA, rankB){
+    return 1/(1 + Math.pow(10, (rankB-rankA)/400));
+  },
+  adjusted: function(rankA, rankB, didWin){
+    var sValue = didWin ? 1 : 0;
+    return rankA + Elo.K * (sValue - Elo.expected(rankA, rankB));
+  },
+  expectedMatch: function(match){
+    return Elo.expected(match.get('playerA').get('person').get('rank'),
+                        match.get('playerB').get('person').get('rank'));
+  },
+  adjustMatchResults: function(match){
+    if (!match) return null;
+    var personA = match.get('playerA').get('person');
+    var rankA = personA.get('rank');
+    var personB = match.get('playerB').get('person');
+    var rankB = personB.get('rank');
+    var playerAdidWin = match.get('winner') == 'playerA';
```

Comment:
I prefer `playerADidWin` over `playerAdidWin` (was trying to figure out what "adid" was). Just nitpicking, you decide